packages/composites/crypto-volatility-index/src/config/index.ts

```diff
@@ -1,2 +1,13 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
 export const DEFAULT_NETWORK = 'ETHEREUM'
 export const NAME = 'CRYPTO_VOLATILITY_INDEX'
+export const DEFAULT_ENDPOINT = 'volatility'
```

Comment:
I think this endpoint name is misleading - suggest naming it "index"

Reply:
we have 2 files in endpoint folder. The `index.ts` that exports all the methods defined in another file where we have the logic of the endpoints. So how about I call it `indexVolatility` ? to keep that file structure of `index.ts` to export everything.

Reply:
Oooh, I see. Let's call it `volatilityIndex` then